model/kubernetes/kubernetes_cluster.rb

```diff
@@ -0,0 +1,43 @@
+#  frozen_string_literal: true
+
+require_relative "../../model"
+
+class KubernetesCluster < Sequel::Model
+  one_to_one :strand, key: :id
+  one_to_one :api_server_lb, class: :LoadBalancer, key: :id, primary_key: :api_server_lb_id
+  many_to_one :private_subnet
+  many_to_many :vms do |ds|
```

Comment:
k8s cluster has vms. Also nodepool has vms. I assume `cluster.vms` doesn't include node's vms, and these are cp ones. Should rename it as `cp_vms`?